visualization/app/codeCharta/ui/ribbonBar/ribbonBar.e2e.ts

```diff
@@ -1,22 +1,18 @@
 import { goto } from "../../../puppeteer.helper"
 import { RibbonBarPageObject } from "./ribbonBar.po"
 import { SearchPanelPageObject } from "../searchPanel/searchPanel.po"
-import { UploadFileButtonPageObject } from "../toolBar/uploadFilesButton/uploadFilesButton.po"
-import { AreaSettingsPanelPageObject } from "./areaSettingsPanel.po"
 
 //Commented out flaky test
 
 describe("RibbonBar", () => {
 	let searchPanel: SearchPanelPageObject
-	let uploadFilesButton: UploadFileButtonPageObject
 	//let searchPanelModeSelector: SearchPanelModeSelectorPageObject
 	let ribbonBar: RibbonBarPageObject
 	//let metricChooser: MetricChooserPageObject
 	//let mapTreeViewLevel: MapTreeViewLevelPageObject
```

Comment:
Do you think we can remove these comments? I don't see anyone fixing it at the moment.

Reply:
you are welcome to do it :) It is not related to migration, so I skipped it.